CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file, in reverse chronological order by release.
 
+## 1.0.0 - TBD
+
+### Added
+
+- [#9](https://github.com/zendframework/zend-expressive-authentication-zendauthentication/pull/9)
+  introduces the usage of zend-expressive-authentication ^ 1.0.
```

Comment:
Remove space after `^` before the version.

Reply:
Done, thanks!